test/evergreen.yml

```diff
@@ -5530,6 +5570,10 @@ tasks:
       - func: "upload stats to evergreen"
         vars:
           test-name: cache_dirty_trigger_read-10_write-90.py
+      - func: "upload artifact"
```

Comment:
These upload statements are repeated in every test. Is there any way to avoid these duplicate statements?
How about creating a function of workgen_upload and pass these statements there?